styles/globals.css

```diff
@@ -2,15 +2,35 @@ html,
 body {
   padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: Roboto, serif;
```

Comment:
Are you sure you want a serif font as fallback when Roboto is not loading?

https://en.wikipedia.org/wiki/Serif